### PR TITLE
rkt: Minor refactoring.

### PR DESCRIPTION
**Pull request by yifan-gu**
Polished a little bit more on the format. cc @kubernetes/sig-rktnetes @pwittrock @johndmulhausen

### CODE REVIEW

File: docs/getting-started-guides/rkt/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This document describes how to run Kubernetes using [rkt](https://github.com/cor
 
 - The [rkt API service](http://coreos.com/rkt/docs/latest/subcommands/api-service.html) must be running on the node.
 
-### Setup network
+### Setup Network
```
Comment (johndmulhausen) [inline docs/getting-started-guides/rkt/index.md]: Actually, sentence casing is correct for headings. Title Casing Like This is only for page titles.

File: docs/getting-started-guides/rkt/index.md [hunk 2]
```diff
@@ -75,10 +75,9 @@ For more information on flannel configuration, please read [CNI/flannel README](
 ##### Use Google Compute Engine (GCE) network
 
 Each VM on GCE has an additional 256 IP addresses routed to it, so it is possible to forego flannel in smaller clusters.
-This can most easily be done by using the builtin kubenet plugin.
-This can be done by setting `--network-plugin=kubenet`.
+This can most easily be done by using the builtin kubenet plugin, by setting the kubelet flag `--network-plugin=kubenet`.
 
-### Launch a local cluster
+### Launch a Local Cluster
```
Comment (johndmulhausen) [inline docs/getting-started-guides/rkt/index.md]: Back to sentence casing.

File: docs/getting-started-guides/rkt/index.md [hunk 3]
```diff
@@ -104,9 +103,9 @@ Then we can launch the local cluster using the script:
 $ hack/local-up-cluster.sh
 ```
 
-We are also working on set up rkt as the container runtime for [minikube](https://github.com/kubernetes/minikube/issues/168).
+We are also working on setting up rkt as the container runtime for [minikube](https://github.com/kubernetes/minikube/issues/168).
 
-### Launch a CoreOS/rkt cluster on Google Compute Engine (GCE)
+### Launch a CoreOS/rkt Cluster on Google Compute Engine (GCE)
```
Comment (johndmulhausen) [inline docs/getting-started-guides/rkt/index.md]: Back to sentence casing.

File: docs/getting-started-guides/rkt/index.md [hunk 4]
```diff
@@ -132,12 +131,12 @@ Then you can launch the cluster by:
 $ cluster/kube-up.sh
 ```
 
-### Launch a CoreOS/rkt cluster on AWS
+### Launch a CoreOS/rkt Cluster on AWS
```
Comment (johndmulhausen) [inline docs/getting-started-guides/rkt/index.md]: Back to sentence casing.

File: docs/getting-started-guides/rkt/index.md [hunk 5]
```diff
@@ -132,12 +131,12 @@ Then you can launch the cluster by:
...
 
 `kube-up` for AWS is currently unsupported.
 Instead, we recommend you to refer the [Kubernetes on AWS guide](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) to launch a CoreOS/rkt cluster on AWS.
 
-### Deploy apps to your cluster
+### Deploy Apps to Your Cluster
```
Comment (johndmulhausen) [inline docs/getting-started-guides/rkt/index.md]: Back to sentence casing.

File: docs/getting-started-guides/rkt/notes.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 ---
 
-# Known issues
+# Known Issues
```
Comment (johndmulhausen) [inline docs/getting-started-guides/rkt/notes.md]: Remove this heading altogether; the page title (as defined in the .yml file) will be a big <h1> at the top here.

Comment (yifan-gu) [inline docs/getting-started-guides/rkt/notes.md]: @johanatan Thank you

**Comment by yifan-gu** [comment]
@johndmulhausen Thank you for reviewing :)

FYI, I might need to add another section about rkt stage1 https://github.com/kubernetes/kubernetes/issues/26201#issuecomment-228427151

**Comment by johndmulhausen** [comment]
Thanks, @yifan-gu !

**Pull request merged**